packages/web/src/components/data-entry/InputV2.tsx

```diff
@@ -48,13 +50,14 @@ export const InputV2 = (props: InputV2Props) => {
     onFocus: onFocusProp,
     onBlur: onBlurProp,
     placeholder,
+    helperText,
     ...other
   } = props
 
   const characterCount = value ? `${value}`.length : 0
   const nearCharacterLimit = maxLength && characterCount >= 0.9 * maxLength
   const elevatePlaceholder = variant === InputV2Variant.ELEVATED_PLACEHOLDER
-  const label = required && !elevatePlaceholder ? `${labelProp} *` : labelProp
+  const label = required ? `${labelProp} *` : labelProp
```

Comment:
is this a bugfix?

Reply:
yeah cause elevated placeholder should def have the * when it's expanded and required imo